src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs

```diff
@@ -23,63 +22,64 @@ namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.Base
 
 
         [OneTimeSetUp]
         public void OneTimeSetUp()
         {
-            #if DEBUG
-            var httpClient = new HttpClient()
-            {
-                BaseAddress = new Uri("https://api.externalintegration.payex.com")
-            };
-
-            var config = new ConfigurationBuilder()
-             .AddJsonFile("appsettings.json")
-             .Build();
-
-            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", config["payexTestToken"]);
-            #elif RELEASE
+            IConfigurationRoot configRoot = new ConfigurationBuilder()
+                .SetBasePath(Environment.CurrentDirectory)
+                .AddJsonFile("appsettings.json", true)
+                .AddEnvironmentVariables()
+                .Build();
+
+            var baseAddress = configRoot.GetSection("SwedbankPayConnectionSettings:ApiBaseUrl").Value;
+            var authHeader = configRoot.GetSection("SwedbankPayConnectionSettings:Token").Value;
             var httpClient = new HttpClient()
             {
-                BaseAddress = new Uri(Environment.GetEnvironmentVariable("Payex.Api.Url", EnvironmentVariableTarget.User))
+                BaseAddress = new Uri(baseAddress)
             };
-            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", Environment.GetEnvironmentVariable("Payex.Api.Token", EnvironmentVariableTarget.User));
-            #endif
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", authHeader);
 
             SwedbankPayClient = new SwedbankPayClient(httpClient);
         }
 
 
         protected OrdersPage GoToOrdersPage(Product[] products, PayexInfo payexInfo, Checkout.Option checkout = Checkout.Option.Anonymous)
         {
+            ThankYouPage page = null;
+
             switch (payexInfo)
             {
                 case PayexCardInfo cardInfo:
 
-                    return PayWithPayexCard(products, cardInfo, checkout).Header.Orders.ClickAndGo();
+                    page = PayWithPayexCard(products, cardInfo, checkout);
 
+                    break;
+                        
                 case PayexSwishInfo swishInfo:
 
-                    return PayWithPayexSwish(products, swishInfo, checkout).Header.Orders.ClickAndGo();
+                    page = PayWithPayexSwish(products, swishInfo, checkout);
+
+                    break;
 
                 case PayexInvoiceInfo invoiceInfo:
 
-                    return PayWithPayexInvoice(products, invoiceInfo, checkout).Header.Orders.ClickAndGo();
+                    page = PayWithPayexInvoice(products, invoiceInfo, checkout);
+
+                    break;
             }
 
-            return null;
+            return page
+                .ThankYou.IsVisible.WaitTo.Within(15).BeTrue()
+                .Header.Orders.ClickAndGo();
+            ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)